homeassistant/components/lock/homematic.py

```diff
@@ -34,7 +34,10 @@ class HMLock(HMDevice, LockDevice):
     @property
     def is_locked(self):
         """Return true if the lock is locked."""
-        return self._hmdevice.is_locked()
+        try:
+            return not self._hm_get_state()
+        except TypeError:
+            return False
```

Comment:
simple: `return not bool(self._hm_get_state())`

Reply:
@pvizeli Okay I simplified the statement.